tests/kubectl_test.go

```diff
@@ -140,4 +144,69 @@ var _ = Describe("[sig-compute]oc/kubectl integration", func() {
 
 	})
 
+	FDescribe("VM instance migration", func() {
+		var virtClient kubecli.KubevirtClient
+
+		runVMIAndExpectLaunch := func(vmi *v1.VirtualMachineInstance, timeout int) *v1.VirtualMachineInstance {
+			return tests.RunVMIAndExpectLaunchWithIgnoreWarningArg(vmi, timeout, false)
+		}
+
+		BeforeEach(func() {
+			virtClient, err = kubecli.GetKubevirtClient()
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		Context("'kubectl get vmim'", func() {
+			It("print the expected columns and their corresponding values", func() {
+				vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros))
+				tests.AddUserData(vmi, "cloud-init", "#!/bin/bash\necho 'hello'\n")
+
+				By("Starting the VirtualMachineInstance")
+				vmi = runVMIAndExpectLaunch(vmi, tests.MigrationWaitTime)
+
+				By("Checking that the VirtualMachineInstance console has expected output")
+				Expect(console.LoginToCirros(vmi)).To(Succeed())
+
+				By("creating the migration")
+				migration := tests.NewRandomMigration(vmi.Name, vmi.Namespace)
+
+				var migrationCreated *v1.VirtualMachineInstanceMigration
+				By("starting migration")
+				Eventually(func() error {
+					migrationCreated, err = virtClient.VirtualMachineInstanceMigration(migration.Namespace).Create(migration)
+					return err
+				}, tests.MigrationWaitTime, 1*time.Second).Should(Succeed(), "migration creation should succeed")
+				migration = migrationCreated
+
+				tests.ExpectMigrationSuccess(virtClient, migration, tests.MigrationWaitTime)
+
+				k8sClient := tests.GetK8sCmdClient()
+				result, _, err := tests.RunCommand(k8sClient, "get", "vmim", migration.Name)
```

Comment:
Maybe we should write a wrapper around this. (not necessary in this PR of course :) )

Reply:
@xpivarc Sure. Do you mean to wrap the `RunCommand` retires or to wrap also the `ExpectMigrationSuccess`?

Reply:
Just the retry. It's all over the place now and it would be safer to have it contained.

Reply:
@xpivarc Sure. Got you. Lets do it in the follow up PR